parachain/runtime/kintsugi/src/xcm_config.rs

```diff
@@ -229,24 +286,26 @@ pub type LocalAssetTransactor = MultiCurrencyAdapter<
     DepositToAlternative<TreasuryAccount, Tokens, CurrencyId, AccountId, Balance>,
 >;
 
+fn general_key_of(id: CurrencyId) -> Junction {
+    let encoded = id.encode();
+    let mut data = [0u8; 32];
+    if encoded.len() > 32 {
```

Comment:
Is this ever possible?

Reply:
Not with our current CurrencyId type, no. But I figured it'd be good to have the panicing point be explicit, rather than hidden inside the copy_from_slice